src/App.jsx

```diff
@@ -1,33 +1,35 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import './App.css';
+
+const todoList = [
+  {
+    title: 'Read pages 1 through 37 of textbook',
+    id: 1
+  },
+  {
+    title: 'Watch tutorial videos',
+    id: 2
+  },
+  {
+    title: 'Complete assignment',
+    id: 3
+  },
+];
 
 function App() {
-  const [count, setCount] = useState(0)
+  const listItems = todoList.map(item => {
```

Comment:
Something to consider: Move this mapping directly in the JSX in the return statement instead of defining it here.

It's a good idea to keep your JSX code inside the return statement of the component, this will make is easier to understand the hierarchy of the webpage.